realm/src/main/java/io/realm/RealmMigration.java

```diff
@@ -17,7 +17,35 @@
 package io.realm;
 
 /**
- * RealmMigration is used to allow user defined migration of one model to a another.
+ * The RealmMigration class is used to describe the migration of one Realm schema to another.
```

Comment:
I don't think so. The RealmMigration class will probably stay as it is, it is just the dynamic API used inside the migration that will change.